server/src/main/java/org/apache/cassandra/sidecar/acl/AuthCache.java

```diff
@@ -46,7 +46,7 @@
  */
 public abstract class AuthCache<K, V>
 {
-    private static final Logger LOGGER = LoggerFactory.getLogger(AuthCache.class);
+    protected final Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
I would prefer a dedicated LOGGER declared per class, from the perspective of clarity. I found it very hard to see the usage of logger that is defined in a superclass.

Reply:
How can you leverage the logger of the implementing class though?

Reply:
right now the limitation we have is that all logs are logged by the AuthCache logger. But we have multiple implementations and there's no way to differentiate between the loggers.

Reply:
Yifan suggested adding another logger in child class if there is need.

Reply:
We still have `getClass()` in actual logger `private final Logger logger = LoggerFactory.getLogger(this.getClass());` just changed access modifier